pom.xml

```diff
@@ -108,6 +108,30 @@
         <module>worker-tracking-report</module>
     </modules>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.hibernate</groupId>
+                <artifactId>hibernate-validator</artifactId>
+                <version>5.2.5.Final</version>
+            </dependency>
+            <dependency>
+                <groupId>io.dropwizard</groupId>
+                <artifactId>dropwizard-bom</artifactId>
+                <version>1.3.2</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>com.github.cafapi</groupId>
+                <artifactId>caf-common</artifactId>
+                <version>1.16.0-SNAPSHOT</version>
```

Comment:
Never reference the SNAPSHOT version unless you're using something from it.  By referencing it you are saying that the latest released version is no good for you, that this new SNAPSHOT version has functionality that you need, and that it will have to be released before you are able to release this.  I know that it not true in this case since nothing has changed in `caf-common` since it's last release.